lib/provider/aws/create-request.js

```diff
@@ -27,7 +28,7 @@ function requestBody(event) {
 
 module.exports = (event, options) => {
   const method = event.httpMethod;
-  const query = event.multiValueQueryStringParameters || event.queryStringParameters;
+  const query = event.multiValueQueryStringParameters || lodash.merge(event.queryStringParameters, event.queryString)
```

Comment:
What do you expect the merge of these to be? isn't event.queryString a string and queryStringParameters an object?